primary-maternal-care/7.md

```diff
@@ -1,14 +1,14 @@
 ---
 book: Primary Maternal Care
-title: 7. Medical problems during pregnancy and the puerperium
+title: . Medical problems during pregnancy, labour and the puerperium
```

Comment:
@nkoeries Please add back the chapter number before the full stop at the start of the title here.